src/artemis/external_interaction/callbacks/rotation/zocalo_callback.py

```diff
@@ -0,0 +1,36 @@
+from __future__ import annotations
+
+from bluesky.callbacks import CallbackBase
+
+from artemis.external_interaction.callbacks.rotation.ispyb_callback import (
+    RotationISPyBHandlerCallback,
+)
+from artemis.external_interaction.exceptions import ISPyBDepositionNotMade
+from artemis.external_interaction.zocalo.zocalo_interaction import ZocaloInteractor
+from artemis.log import LOGGER
+
+
+class RotationZocaloHandlerCallback(CallbackBase):
```

Comment:
Should: A bit of a docstring would be good, this is probably vaguely user facing